src/main/java/org/mifos/connector/channel/api/implementation/GSMATransactionApiController.java

```diff
@@ -25,11 +25,13 @@ public class GSMATransactionApiController implements GSMATransactionApi {
     private Logger logger = LoggerFactory.getLogger(this.getClass());
 
     @Override
-    public GsmaP2PResponseDto gsmatransaction(GsmaTransfer requestBody, String correlationId,String amsName,String accountHoldId) throws JsonProcessingException {
+    public GsmaP2PResponseDto gsmatransaction(GsmaTransfer requestBody, String correlationId,String amsName,String accountHoldId,String reconciled,String mpesaTxnId) throws JsonProcessingException {
         Headers headers = new Headers.HeaderBuilder()
                 .addHeader("X-CorrelationID", correlationId)
                 .addHeader("amsName",amsName)
                 .addHeader("accountHoldingInstitutionId",accountHoldId)
+                .addHeader("reconciled",reconciled)
+                .addHeader("mpesaTxnId",mpesaTxnId)
```

Comment:
remove these headers from channel connector

Reply:
This is needed further in mpesa- connector for settlement flow.